server/app/services/languageProcessor/TaskMap.java

```diff
@@ -95,13 +118,23 @@ public JsonNode getTicketAssignee(String issueKey, JsonNode responseBody) {
    */
   public static JsonNode parseToJson(String status, String message) {
```

Comment:
Sorry, I think just rename this to `botResponseToJson` is fine. Still pass in `status` and `message` as parameter.

But consider make this accessible from other places, e.g. `QueryHandler` can use this method without doing `BotResponse botReponse = new BotResponse()` over there.